### DISCUSSION TITLE
Auto-level offset auto-setup

### DISCUSSION THREAD

**Discussion by Richrfl** [FrSkyRC/ETHOS-Feedback-Community#2894] [Ideas]
I just found out one feature that for the first time Spektrum introduced before FrSky, which I have seen on iNav and/or Aurdupilot before, which is an auto-level offset self-setting, which, in the case of FrSky, which is not embedded in any model from the factory, may need some consideration. Of course, this is applicable to all gyro-embedded receivers (I guess Series S?).
The deal would be to enable the feature (auto offset set up) with a switch, and while enabled, fly the model straight and flat for a few seconds, while the Gyro filters and sets the offsets, and auto disables when the offsets are set, avoiding any issue regarding on when the pilot switches off the process.

**Comment by Crunch69** [comment]
instant trim?

**Comment by Richrfl** [comment]
No, instant trim is not related to gyro operation. Specifically when you have a gyro receiver, like the SR10, you run a level check for the gyro in the bench, so that when set your switch for Auto Level, the gyro will level up the plane. However, in the air, the level flight longitudinal axis is close, but it is not the same that you have in the bench, therefore, in Auto Level, the plane will either go up or down, as the gyro does not use the altimeter to control level. Therefore, a control switch that will re-adjust the gyro to actual level flight is what I am talking about.
